tests/test_web/test_module.py

```diff
@@ -224,50 +224,169 @@ async def test_keys_properly_propagated(display: DisplayFixture):
     GridLayout = reactpy.web.export(module, "GridLayout")
 
     await display.show(
-        lambda: GridLayout({
-            "layout": [
-                {
-                    "i": "a",
-                    "x": 0,
-                    "y": 0,
-                    "w": 1,
-                    "h": 2,
-                    "static": True,
-                },
-                {
-                    "i": "b",
-                    "x": 1,
-                    "y": 0,
-                    "w": 3,
-                    "h": 2,
-                    "minW": 2,
-                    "maxW": 4,
-                },
-                {
-                    "i": "c",
-                    "x": 4,
-                    "y": 0,
-                    "w": 1,
-                    "h": 2,
-                }
-            ],
-            "cols": 12,
-            "rowHeight": 30,
-            "width": 1200,
-        },
+        lambda: GridLayout(
+            {
+                "layout": [
+                    {
+                        "i": "a",
+                        "x": 0,
+                        "y": 0,
+                        "w": 1,
+                        "h": 2,
+                        "static": True,
+                    },
+                    {
+                        "i": "b",
+                        "x": 1,
+                        "y": 0,
+                        "w": 3,
+                        "h": 2,
+                        "minW": 2,
+                        "maxW": 4,
+                    },
+                    {
+                        "i": "c",
+                        "x": 4,
+                        "y": 0,
+                        "w": 1,
+                        "h": 2,
+                    },
+                ],
+                "cols": 12,
+                "rowHeight": 30,
+                "width": 1200,
+            },
             reactpy.html.div({"key": "a"}, "a"),
             reactpy.html.div({"key": "b"}, "b"),
             reactpy.html.div({"key": "c"}, "c"),
         )
     )
 
-    parent = await display.page.wait_for_selector(".react-grid-layout", state="attached")
+    parent = await display.page.wait_for_selector(
+        ".react-grid-layout", state="attached"
+    )
```

Comment:
Given this random formatting change, I'm assuming you're using the `black` formatter?

This repo uses the `ruff` formatter, which we have pre-configured within the ReactPy project files.

Reply:
Looks like you're running `hatch fmt` from a system-wide python interpreter, instead of a `venv`. Not sure how that would cause issues, but it might.

Reply:
It's alright, I'll just go ahead and close this comment. Our style checking workflow isn't bothered by the change so it doesn't seem to matter to Ruff.